packages/web_benchmarks/lib/src/runner.dart

```diff
@@ -343,3 +339,33 @@ class BenchmarkServer {
     }
   }
 }
+
+/// Compilation options for bulding a Flutter web app.
+///
+/// This object holds metadata that is used to determine how the benchmark app
+/// should be built.
+class CompilationOptions {
+  /// Creates a [CompilationOptions] object.
+  const CompilationOptions({
+    this.renderer = WebRenderer.html,
```

Comment:
The default should ABSOLUTELY be `canvaskit`. I'd say ditch the `html` option entirely!

Reply:
Well, this is published. Maybe just default to `canvaskit`